docs/src/modules/components/LocalizationTable.js

```diff
@@ -29,26 +26,29 @@ const Root = styled('div')(({ theme }) => ({
       border: `1px solid`,
       borderColor: (theme.vars || theme).palette.error.light,
     },
+    ...theme.applyStyles('dark', {
```

Comment:
I was curious about whether we could use `applyStyles()` or not. It was added with https://github.com/mui/material-ui/pull/40667, released with @mui/material@5.15.7, we have this dependency:

https://github.com/mui/mui-x/blob/c0b599300f01c54a96a4443a6296b10c6555036d/packages/x-data-grid/package.json#L61

since https://github.com/mui/mui-x/pull/12516, v7.0.0, so it looks likes yes. Cool

Reply:
Yes, I was pretty sure we can't use it 😅
But it turns out the minimum supported version guarantees that `applyStyles` will always be available 👍🏻